services/auth/basic.go

```diff
@@ -23,6 +23,9 @@ var (
 	_ Named  = &Basic{}
 )
 
+// BasicMethodName is the constant name of the basic authentication method
+const BasicMethodName = "basic"
```

Comment:
`BasicAuthMethodName` is more accurate.

Reply:
But the package is called auth and the reverseproxy constant is also `ReverseProxyMethodName`